internal/utils/get_distro.go

```diff
@@ -32,8 +33,9 @@ Error if any
 func GetOSInfo() (distro DistroInfo, err error) {
 	// Get the architecture
 	distro.Architecture = runtime.GOARCH
+	file := "/etc/os-release"
```

Comment:
May not work out for a few OS but maybe add a TODO over here to handle this in the future.. Keep it like this for now

Reply:
Yes, you are right. That's why on the comment description of this function it says this is only for Linux (not sure if on darwin this works). For the moment I don't plan to install directly the dependencies on windows, and for sure I need to test this on macOS.